lib/tesla_api/client.rb

```diff
@@ -77,11 +77,11 @@ def expired?
     end
 
     def get(url)
-      api.get(url.sub(/^\//, ''), nil, { 'Authorization' => "Bearer #{access_token}" }).body
+      api.get(api_uri + url.sub(/^\//, ''), nil, { 'Authorization' => "Bearer #{access_token}" }).body
     end
 
     def post(url, body: nil)
-      api.post(url.sub(/^\//, ''), body, { 'Authorization' => "Bearer #{access_token}" }).body
+      api.post(api_uri + url.sub(/^\//, ''), body, { 'Authorization' => "Bearer #{access_token}" }).body
```

Comment:
This won't work. The Faraday client (`@api`) above already prepends the base URI provided to it.

Reply:
I resolved this with my updated commit.